components/divider/index.css

```diff
@@ -10,46 +10,110 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+/* divider/index.css
+ *
+ *
+ */
 
+.spectrum-Divider {
+  /* default thickness no size */
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-medium);
+
+  /* default background color no size */
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-medium);
+
+  /* background colors */
+  --spectrum-divider-background-color-small: var(--spectrum-gray-300);
+  --spectrum-divider-background-color-medium: var(--spectrum-gray-300);
+  --spectrum-divider-background-color-large: var(--spectrum-gray-800);
+
+  /* static white background colors */
+  --spectrum-divider-background-color-small-static-white: var(--spectrum-transparent-white-300);
+  --spectrum-divider-background-color-medium-static-white: var(--spectrum-transparent-white-300);
+  --spectrum-divider-background-color-large-static-white: var(--spectrum-transparent-white-800);
+
+  /* static black background colors */
+  --spectrum-divider-background-color-small-static-black: var(--spectrum-transparent-black-300);
+  --spectrum-divider-background-color-medium-static-black: var(--spectrum-transparent-black-300);
+  --spectrum-divider-background-color-large-static-black: var(--spectrum-transparent-black-800);
+}
+
+/* small divider */
 .spectrum-Divider--sizeS {
-  @remapvars {
-    find: --spectrum-divider-s-;
-    replace: --spectrum-divider-;
-  }
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-small);
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-small);
 }
 
+/* medium divider */
 .spectrum-Divider--sizeM {
-  @remapvars {
-    find: --spectrum-divider-m-;
-    replace: --spectrum-divider-;
-  }
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-medium);
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-medium);
 }
 
+/* large divider */
 .spectrum-Divider--sizeL {
-  @remapvars {
-    find: --spectrum-divider-l-;
-    replace: --spectrum-divider-;
-  }
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-large);
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-large);
 }
 
-.spectrum-Divider {
-  --spectrum-divider-vertical-height: 100%;
-}
 
 .spectrum-Divider {
+  block-size:  var(--mod-divider-thickness,
+                  var(--spectrum-divider-thickness));
   inline-size: 100%;
-  block-size: var(--spectrum-divider-height);
 
   /* Show the overflow for hr in Edge and IE. */
   overflow: visible;
 
   border: none;
-  border-width: var(--spectrum-divider-height);
-  border-radius: var(--spectrum-divider-height);
-}
+  border-width: var(--mod-divider-thickness,
+                    var(--spectrum-divider-thickness));
+  border-radius:  var(--mod-divider-thickness,
+                      var(--spectrum-divider-thickness));
+
+  background-color: var(--mod-divider-background-color,
+                        var(--spectrum-divider-background-color));
+
+  /* static white variant colors */
+  &--staticWhite {
```

Comment:
hmm, this looks like a repeat. would it make sense to specify this for all `--staticWhite` and a separate one for black variants?

Reply:
I'm not sure I follow. Can you show me an example of what you mean? Thanks!

Reply:
Ah - running into the same thing in my Switch work.
Yes, we pretty much have to re-define those vars with the same syntax for static white and black, only changing the var names.
Given that we don't re-map vars anymore I think this is ok.